graphql/src/store/query.rs

```diff
@@ -238,14 +251,15 @@ fn build_fulltext_order_by_from_object(
 /// Parses GraphQL arguments into a EntityOrder, if present.
 fn build_order_direction(
     arguments: &HashMap<&q::Name, q::Value>,
-) -> Result<Option<EntityOrder>, QueryExecutionError> {
+) -> Result<OrderDirection, QueryExecutionError> {
     Ok(arguments
         .get(&"orderDirection".to_string())
-        .and_then(|value| match value {
-            q::Value::Enum(name) if name == "asc" => Some(EntityOrder::Ascending),
-            q::Value::Enum(name) if name == "desc" => Some(EntityOrder::Descending),
-            _ => None,
-        }))
+        .map(|value| match value {
+            q::Value::Enum(name) if name == "asc" => OrderDirection::Ascending,
+            q::Value::Enum(name) if name == "desc" => OrderDirection::Descending,
+            _ => OrderDirection::Ascending,
```

Comment:
Should it not be an error if the `orderDirection` is present but something we don't recognize?

Reply:
I just followed what the code did before - the old `None` really ended up as `EntityOrder::Ascending` somewhere in the guts of the machinery. There's good reason to make that an error, as you say, but I didn't want to mess with that in this PR.

Reply:
Make sense to separate the changes, but now that we noticed it's a good opportunity to try to fix this, even if it risks breaking queries in the wild. Could you make a follow up PR for that?